src/renderer/tests/basic.rs

```diff
@@ -53,6 +53,19 @@ fn render_variable_block_lit_expr() {
     }
 }
 
+#[test]
+fn render_variable_block_lit_concat_expr() {
+    let inputs = vec![
+        ("{{ \"music\" ~ \" start\" }}", "music start"),
+        ("{{ (4 + 4) ~ \"88\" }}", "888"),
```

Comment:
that's precisely something that is an issue I think, we want that kind of concatenation to be an error